docs/source/releases/v1_13_0a0.rst

```diff
@@ -16,6 +16,7 @@ Version 1.13.0a0 (2024-04-24)
 * Enhancements
 
   * Initial Stage of the GeoIPS CLI
+  * CLI Stage Two - new implementation of 'geoips run' and more args for 'geiops list'
```

Comment:
I would update the title of this PR to reflect these actual changes,  rather than just CLI stage 2. 

Also,  were the bits about editable mode just added error checking?  Could we get the same results by just checking if the tests directory exists ? (Ie, I don't think it's specifically that the package is installed in editable mode,  just that the full repository has been cloned to disk ?)

Reply:
After reviewing, I think this is good as-is. One quibble I have is that the same logic is used in four different locations to raise an appropriate error when a command that requires editable mode is called. I think it's fine for now, but I think we should go through the tests at some point to trim them down and reduce code duplication.